db/migrate/20221018180029_add_user_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] This references should point to the `author_id`, not `user_id`. Kindly run another migration to fix this.

  You can visit this thread for cues: [https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration](https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration)

Reply:
Done